docs/wiki/Home.md

```diff
@@ -14,20 +14,20 @@ The following instructions assume that you have already forked **dcrdex**.
 
 1. Initialize your forked wiki on github by navigating to the wiki tab of your forked **dcrdex** repo and clicking the "Create the first page" button. Be sure to "Save page" as well.
 
-<img src="images/wiki-creation.png" width="700">
+    <img src="images/wiki-creation.png" width="700">
 
 2. In your terminal, navigate to your local **dcrdex** directory and add your remote wiki.
 
-```
-git remote add wiki https://github.com/your-username/dcrdex.wiki.git
-git push wiki -d master
-git subtree push --prefix docs/wiki wiki master
-```
+    ```text
+    $ git remote add wiki https://github.com/your-username/dcrdex.wiki.git
+    $ git push wiki -d master
+    $ git subtree push --prefix docs/wiki wiki master
```

Comment:
There's actually a markdownlint rule we've ignored that says not to use `$` without showing output.  While I don't think we need to show output (rule ignored), I also don't think we need the `$` in every case unless it is really needed to underscore that the commands are typed and it's obvious and explicit here.  Also this can be sh instead of text.